### DISCUSSION TITLE
Webp support

### DISCUSSION THREAD

**Discussion by aleontbi** [ImageMagick/ImageMagick#4340] [Help]
Hi everyone 👋
I have problems with my server because not support webp. Do you have any idea why it doesn't work?
Thanks a lot!

![Captura de Pantalla 2021-10-08 a la(s) 11 21 10](https://user-images.githubusercontent.com/92115063/136590638-b9687b82-9569-4990-9392-c2db18237a96.png)
![Captura de Pantalla 2021-10-08 a la(s) 11 23 37](https://user-images.githubusercontent.com/92115063/136590943-e720267c-ad67-484d-9532-65ac872a8442.png)

**Comment by fmw42** [comment]
Webp (libwebp) must be installed as a delegate in ImageMagick for it to work with Imagick.  Just having the entries for the delegate in the delegates.xml file only says how to use it if it is installed. It does not mean that is was installed.

Check either or both

`convert -version`

to see if it lists webp

or 

`convert -list format`

and look to see that you have an entry for it such as

`WEBP* WEBP      rw+   WebP Image Format (libwebp 1.2.1 )`

Also be sure that Imagick is using the same version of ImageMagick, if you have webp installed.

**Reply by aleontbi** [reply]
@fmw42  I executed `convert -list format` but webp doesn't appear. Should I install libwebp?

![Captura de Pantalla 2021-10-08 a la(s) 11 54 11](https://user-images.githubusercontent.com/92115063/136594528-95056531-ebba-432b-85b8-a4111e70662f.png)

**Reply by fmw42** [reply]
Yes. You need to install it and then be sure ImageMagick can see it. By the same command.
